setup.cfg

```diff
@@ -1,4 +1,4 @@
 [flake8]
 exclude = examples, envisage/ui/single_project
 ignore = E266, W503
-per-file-ignores = */api.py:F401
+per-file-ignores = */api.py:F401, */heartbeat.py:H101, */kernelapp.py:H101, setup.py:H102,
```

Comment:
are these all specifically related to this PR?

Reply:
yes. H101 and H102 are the copyright header errors and heartbeat.py, kernelapp.py and setup.py are excluded because 

- `heartbeat.py` and `kernelapp.py` are borrowed/adapted from the IPython project. (I'm not sure if that means we can just add our copyright header on top of the IPython copyright header)
- `setup.py` generates a `version.py` file during installation the file has two copies of the copyright header.

Reply:
a total of 4 errors defined by `copyright_header.py` https://github.com/enthought/envisage/blob/26fd5196a587a14d209829159d61dd3df431565f/copyright_header/copyright_header.py#L97-L142

Reply:
I understand now

Reply:
For stuff from IPython, if we have modified the files then we have copyright to the changes, so the copyright header should include both.  We want to be careful about not asserting a copyright date prior to our first modification, so 2005-2020 is probably not right.